game_patch/object/obj_light.cpp

```diff
@@ -133,6 +134,25 @@ ConsoleCommand2 mesh_static_lighting_cmd{
     "Toggle mesh static lighting calculation",
 };
 
+CallHook<void(rf::Entity&)> entity_update_muzzle_flash_light_hook{
+    0x0041E814,
+    [](rf::Entity& ep) {
+        if (g_game_config.muzzle_flash) {
```

Comment:
it doesn't remove muzzle flash completely right? only the light decal on walls? The bitmap at the muzzle stays? Perhaps it would make sense to keep calling it consistently muzzle flash light then and not shortening to muzzle flash. 
But if it's "accessability feature" shouldn't it disable also the flash bitmap? I assume it's for people who don't like fast light changes

Reply:
> it doesn't remove muzzle flash completely right? only the light decal on walls?

I considered whether it made sense to disable both, but I think it would make the option far too much of a disadvantage to use (since it would be far more difficult to know when enemies were shooting). Furthermore, removing muzzle flash bitmaps can be done easily with clientside mods, so I'm not concerned about it - this PR is designed to address the part that players don't already have the ability to do through other means.

Unlike the muzzle flash bitmap, the blinking dynamic light is extremely obnoxious and can't currently be avoided without putting yourself at a noted disadvantage (due to not seeing amped players/players with flag through geometry). This PR fixes that.